src/server/update.rs

```diff
@@ -0,0 +1,250 @@
+use log::{debug, error, info};
+
+use std::{fs, io::ErrorKind, path::Path, process::exit};
+
+use crate::{
+  constants, files::config::load_config, server, steamcmd::steamcmd_command, utils::get_working_dir,
+};
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct UpdateInfo {
+  current_buildid: String,
+  latest_buildid: String,
+}
```

Comment:
Nit, would it be possible to add an underbar between build and id for these two? 
`current_build_id`
`latest_build_id`

Reply:
Sounds good to me!